File: fibonacci.py
import sys

def main():
    index = int(sys.argv[1])
    assert_index(index)
    print(str(fibonacci(index)))

def fibonacci(index):
    if index < 3:
        return 1
    return fibonacci(index - 1) + fibonacci(index - 2)

def assert_index(index):
    if index < 1 or index > 127:
        raise Exception("Index must be a positive integer between 1 and 127.")

main()
    

File: fibonacci.js
function main() {
    index = parseInt(process.argv[2])
    assertIndex(index)
    console.log(fibonacci(index))
}

function fibonacci(index) {
    const sequence = []
    sequence[0] = BigInt(0)
    sequence[1] = BigInt(1)
    sequence[2] = BigInt(1)
    
    for (let i = 3; i <= index; i++) {
        sequence[i] = sequence[i - 1] + sequence[i - 2]
    }

    return sequence[index].toString()
}

function assertIndex(index) {
    if (index < 1 || index > 99999) {
        throw new Error('Index must be a positive integer between 1 and 99999.')
    }
}

main()